backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+= Introduction to {oadp-full}
+include::_attributes/common-attributes.adoc[]
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+
+The {oadp-first} product safeguards customer applications on {product-title}. OADP provides disaster recovery protection for {product-title} applications, application-related cluster resources, persistent volumes, and internal images.
+
+OADP can be used to back up containerized applications and virtual machines (VMs).
+
+OADP cannot be a disaster recovery solution for etcd or OpenShift Operators. For more information on etcd, see xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[Backing up etcd].
+
+OADP provides the following APIs to enable multiple approaches to customize a backup to avoid backing up inappropriate resources by using namespaces or labels.
```

Comment:
+1 to Steven's comment. I am not sure that adding links will clarify the sentence. I find the infinitive phrases difficult to follow. Is this sentence is describing one, very specific action that these APIs enable, or if it is a list of capabilities that these APIs share? Do the APIs use namespaces and labels to avoid backing up the wrong resources? 

I understand that some of this content is a placeholder for future revisions.

Reply:
@michaelryanpeter - I am sorry not repeating myself

Reply:
@anarnold97, I agree with Michael and Steven. I think you should comment out lines 17-26 until you can link to the API information. In this state, it's not providing additional value.

Reply:
I originally had it separated, but the first reviewer wanted a change and suggested adding that sentence. So can I go back to something like the original:
```
There exists numerous approaches to customize a backup to avoid backing up inappropriate resources by using namespaces or labels. 

OADP provides the following APIs:

    Backup

    Restore

    Schedule

    BackupStorageLocation

    VolumeSnapshotLocation
```
These would then link to the new main sections in the document.